src/main/java/org/openelisglobal/dictionary/controller/rest/DictionaryRestController.java

```diff
@@ -145,9 +145,20 @@ public ResponseEntity<?> showNextPreviousDictionary(HttpServletRequest request)
         return ResponseEntity.status(HttpStatus.OK).body(url);
     }
 
+    /**
+     * @param request
+     * @param form
+     * @param result
+     * @param status
+     * @param redirectAttributes
+     * @return
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws NoSuchMethodException
+     */
     @RequestMapping(value = "/Dictionary", method = RequestMethod.POST)
     public ResponseEntity<?> showUpdateDictionary(HttpServletRequest request,
-                                             @ModelAttribute("form") @Valid DictionaryForm form, BindingResult result, SessionStatus status,
+                                                  @RequestBody @Valid DictionaryForm form, BindingResult result, SessionStatus status,
```

Comment:
the issue here was lack of @RequestBody where @RequestBody is used to indicate that a method parameter should be bound to the body of the HTTP request. It tells Spring to `deserialize` the incoming request payload and convert it into a Java object....etc!